plugins/request-uri-manipulations.rst

```diff
@@ -0,0 +1,25 @@
+Request URI Manipulations
+=========================
+
+Request URI manipulations can be done thanks to several plugins:
+
+* ``AddHostPlugin``: Sets default host, scheme and port of each request.
```

Comment:
depending on configuration, this can force the host or set it to a default value. maybe "Set host, scheme and port. Depending on configuration, the host is overwritten in every request or only set if not yet defined in the request."